src/lib/common/string.cpp

```diff
@@ -95,7 +95,6 @@ bool isIPv6(const std::string& in)
 {
   size_t      pos;
   std::string partip;
-  std::string resu;
```

Comment:
Look a little further down.  It is added to and after that not used.
cppcheck seems a bit smarter than the compiler :-)